danielssonn/ethTO
# feat: refactor swing swapper

## Purpose

1. Refactored `SwingSwapper`

## Testing Instructions

1. Confirm the Swing modal behaves as intended (optional):

- Pull down the branch
- Move into the `client` directory and run `npm run dev`
- Navigate to `http://localhost:3000/arrivals`
- Click on Swap tokens

2. Review code

## Checklist
